morpheus/_lib/cmake/libmorpheus.cmake

```diff
@@ -109,6 +108,17 @@ target_include_directories(morpheus
     $<INSTALL_INTERFACE:include>
 )
 
+# Add the include directories of the cudf_helpers_project since we dont want to link directly to it
+get_target_property(cudf_helpers_include ${cudf_helpers_target} INTERFACE_INCLUDE_DIRECTORIES)
+
+target_include_directories(morpheus
+  PRIVATE
+    ${cudf_helpers_include}
+)
+
+# Also add a dependency to the target so that the headers are generated before the target is built
+add_dependencies(morpheus ${cudf_helpers_target})
+
```

Comment:
Not sure, this is also a change inherited from the original `mdd_relocatable-package` branch. 

My guess is that libmorpheus doesn't need to link to the cudf_helpers so because it is a python C-extension which is loaded dynamically by the interpreter when we call `CudfHelper::load()` from our other python modules.

Reply:
Ah you are correct. I think this was part of some changes I was working on where I was trying to remove the need for the cudf_helpers library entirely. However, I couldnt get that to work 100% but this part stuck.